.changeset/dull-plums-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": patch
```

Comment:
i dint think it should be a patch here. minor i think

Reply:
@themooneer I'm not very clear on this TBH. Are we looking at the semver from the perspective of the app user or the other teams ?